test/protobuf/protoc/cli_integration_test.exs

```diff
@@ -70,6 +70,32 @@ defmodule Protobuf.Protoc.CLIIntegrationTest do
       assert descriptor.name == "User"
     end
 
+    test "include_docs option", %{tmp_dir: tmp_dir, proto_path: proto_path} do
```

Comment:
Also please include a test that we don't add docs when `include_docs` isn't set.

Reply:
✔️ Done!